docs/adapting-settings.md

```diff
@@ -126,30 +126,37 @@ The elasticsearch server url used for search indexing.
 ELASTICSEARCH_URL = 'elasticserver:9200'
 ```
 
-## Mongoengine/Flask-Mongoengine options
-
-### MONGODB_HOST
-
-**default**: `localhost`
+RFC-1738 formatted URLs are also supported:
 
-The mongodb hostname used by udata.
-
-### MONGODB_PORT
-
-**default**: `27017`
+```python
+ELASTICSEARCH_URL = 'http://<user>:<password>@<host>:<port>'
+```
 
-The mongodb post used by udata.
+## Mongoengine/Flask-Mongoengine options
 
-### MONGODB_DB
+### MONGODB_SETTINGS
 
-**default**: `udata`
+**default**:
+```python
+MONGODB_SETTINGS = {
+    'host': 'mongodb://localhost:27017/udata'
+}
+```
 
 The mongodb database used by udata.
 During tests, the test database will use the same name suffixed by `-test`
 
 See [the official Flask-MongoEngine documentation][flask-mongoengine-doc]
 for more details.
 
+Authentication is also supported in the URL:
+
+```python
+MONGODB_SETTINGS = {
+    'host': 'mongodb://<user>:<password>@<host>:<port>/<database>'
+}
```

Comment:
I think for a single parameter the `MONGODB_HOST` is cleaner than a single property in a dict